src/integrationTest/java/uk/gov/hmcts/reform/ccd/data/DeletionScenarios.java

```diff
@@ -520,6 +520,45 @@ protected static Stream<Arguments> provideCaseDeletionScenarios() {
                                                         1504259907351231L, 1504259907351230L,
                                                         1504259907351229L)),
                     Map.of("FT_MasterCaseType", List.of(1504259907351234L,1504259907351228L))
+                ),
```

Comment:
Please add LAU scenario too.

Reply:
Once log and audit exception is thrown , it will throw the exception to higher level and deletion will stop there. It's difficult to add Log and audit exception Integration test